chatroom-socket-server/server.js

```diff
@@ -29,25 +34,24 @@ io.on("connection", (socket) => {
         });
     }
     socket.emit("users", users);
```

Comment:
If this is building a list of users every time someone connects, we might be able to have a global users list that we would just add and remove from on each connect or disconnect